lib/auth/github_authenticator.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require_dependency 'has_errors'
+require 'has_errors'
```

Comment:
Is this required since it is auto-loaded now?

Reply:
It depends where it’s used and if it’s required by a plugin or not as well. I’ll try to remove some `require` and check if it breaks everything or not :grin:

Reply:
I guess no harm leaving this in as well right. Let us ignore this for now :)